src/main/java/beam/physsim/jdeqsim/AgentSimPhysSimInterfaceDebugger.java

```diff
@@ -27,12 +27,13 @@ public void handleEvent(Event event) {
     }
 
     private void exploreAdjacentLinkConnectivity(Event event) {
+        // FIXME Is this used on productioin or only for debug purpose?
```

Comment:
Should this be an issue instead? I know there is an existing issue to clean all these up into issues. So maybe we shouldn't add to the load?

Reply:
This is controlled by `beam.physsim.jdeqsim.agentSimPhysSimInterfaceDebugger.enabled` which is set to `false` by default. So it should be fine to leave this as is because it's debugging tool.